fleet-terraform-generator/internal/module/generate.go

```diff
@@ -120,13 +120,27 @@ func Generate(path, policyTemplateName, dataStreamName, inputName string) (*Terr
 				Default:     &terraform.NullableValue{Value: "default"},
 			},
 		},
+		"fleet_data_stream_description": {
```

Comment:
This looks like it would be a description of the package policy (and hence all of the data streams within the package policy). So I am thinking to a name it `fleet_package_policy_description`.

Reply:
I was staying consistent with ``fleet_data_stream_namespace`` since the generator creates integrations for each data stream and you deploy each fleet policy data stream independently with this setup. Not opposed to naming it ``fleet_package_policy_description`` if you think that makes more sense

Reply:
I was thinking of the `fleet_data_stream_namespace` as being specific to the Elasticsearch data stream (e.g. `logs-foo.bar-{namespace}`).

This and the suffix option both relate to the package policy. So I think they should both begin with `fleet_package_policy_{suffix,description}`.

Reply:
Makes sense, just made this change and updated the integrations. I noticed I accidently included more integrations in a previous commit, so I've reset it back to the expected list for now and regenerated them. I'm running a separate fork that contains all of the modules I'm working with.